src/components/motion/MotionController.cpp

```diff
@@ -31,13 +31,43 @@ bool MotionController::ShouldWakeUp(bool isSleeping) {
   }
   return false;
 }
+
+bool MotionController::ShouldRollDice() {
+  if (z > 0) {
+    diceRollFirstTrigger = false;
+    return false;
+  }
```

Comment:
Sounds good -- definitely can change that.  I'm thinking `WristRolled()` or something similar.

Reply:
I think that `ShouldRollDice` and `ShouldWakeUp` could have its logic somehow combined...

Reply:
That would make sense. Maybe for now this app could use `ShouldWakeUp()`, and in another PR that could be improved?

Reply:
I actually did try using the `ShouldWakeUp` logic and it was way too sensitive for the dice roll.  It was rolling 3-4 times in one motion.

While they're somewhat similar motions, I think they're different enough to warrant separate functions.  The one I made for the dice roll is intended to be much more restrictive to avoid unintentional rolls.

Reply:
@Riksu9000 I updated the name of the function in https://github.com/JF002/InfiniTime/pull/565/commits/42b048d648c6f2aad3064e1760771321f0d08e19

Reply:
Maybe the function could return the "strength" of the roll, and some apps could just have a higher threshold. Then just one function would be enough. It doesn't need to be changed in this PR, but just an idea.